src/Credentials.ts

```diff
@@ -1,9 +1,11 @@
+import type { ICredentials } from './internal/type'
```

Comment:
please add and always add real source file extension

Reply:
https://github.com/minio/minio-js/blob/master/CONTRIBUTING.md#style-guide

This is needed in the source so our build script can build valid package for all users, even esm and `"moduleResolution": "node16"`

Reply:
Sorry, that's my auto-completion that imported this type and omitted the file extension. I'll take a closer look next time to ensure the extension is always included.

That's fixed in a21c956